packages/frontend-shared/src/utils/spec-utils.ts

```diff
@@ -125,7 +126,7 @@ export function useCachedSpecs (specs) {
     const specsAreDifferent =
         currentSpecs.length !== prevSpecs.length ||
         currentSpecs.some(
-          (spec, idx) => spec.absolute !== prevSpecs[idx]?.absolute,
+          (spec, idx) => !_.isEqual(spec, prevSpecs[idx]),
```

Comment:
Ah, this is probably far better than my solution, which was passing a custom function to compare the lists: https://github.com/cypress-io/cypress/pull/20744/files#diff-4227699afbcb444e02cc568cf473652e333a9a2847d0164d00e627061772ddd1R191

When you rebase you can probably eliminate what I did. :facepalm:

Reply:
This could probably be simplified further to just `_.isEqual(currentSpecs, prevSpecs)`... lodash's isEqual is always going to take the optional path (will start with `currentSpecs.length !== prevSpecs.length`, etc)